ibis/common/enums.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABCMeta
+from enum import Enum, EnumMeta
+
+from public import public
+
+from ibis.common.validators import Coercible
+
```

Comment:
Do you have a bettern name for the module than `enums`?

Reply:
They seem fine to me!